sdk/cosmos/azure-cosmos/src/main/java/com/azure/cosmos/implementation/Constants.java

```diff
@@ -30,7 +30,7 @@
         public static final String LAST_MODIFIED = "_ts";
         public static final String COUNT = "_count";
         public static final String E_TAG = "_etag";
-        public static final String AGGREGATE = "_aggregate";
+        public static final String VALUE = "_value";
```

Comment:
Does `_value` represents `_aggregate` queries ?

Reply:
I don't exactly understand your question here. But _aggregate is the keyword that we were appending to value aggregate results and there is no such thing like ```_aggregate``` query. Now I changed it to ```_value``` to be more generic.

Reply:
Thanks for explaining, my concern is, if we are making this more generic, then we should probably change `AggregateDocumentQueryExecutionContext` to `ValueDocumentQueryExecutionContext` or something more generic.

Reply:
No! So there can be both value and non value aggregates, and only value aggregates return scalar values. And appending this key here has nothing to do with the aggregate query as such and its being done just to convert the scalar results to key value pair for our processing convenience because of the way query pipeline handles responses (Documents).